todo_app/app.py

```diff
@@ -1,4 +1,6 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
+from operator import itemgetter
+from todo_app.data.session_items import *
```

Comment:
Minor, but you'll sometimes see opinions that while this way of importing is valid, it's unwise - particularly the risk is "clobbering" methods of the same name, e.g. if someone defined a custom function in `session_items.py` called `itemgetter`, then that would override the earlier import and suddenly this code would break without any changes in this file - so particularly if using it to import external libraries it can lead to slightly mysterious errors!

Generally better is either importing functions by name:
```python
# by name
from todo_app.data.session_items import get_items, add_item # etc.
# used like:
get_items()

# or just by library
import todo_app.data.session_items as session_items
# used like:
session_items.get_items()
```

[Official python doc source - search for `import *` for the relevant bit](https://docs.python.org/3/tutorial/modules.html)